base/sparse/cholmod.jl

```diff
@@ -576,7 +617,7 @@ for Ti in IndexTypes
             d
         end
 
-        function spsolve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
+        function solve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
```

Comment:
It would be great. If you loop through the types and names it is almost the same number of lines of code.